keyboards/gck/num3x3/info.json

```diff
@@ -0,0 +1,60 @@
+{
+    "keyboard_name": "gck/num3x3",
+    "maintainer": "Giliy666",
+    "manufacturer":"GCK",
+    
+    "diode_direction": "COL2ROW",   
+
+    "features": {
+        "command": false,
+        "console": false,   
+        "extrakey": true,
+        "mousekey": false,            
+        "nkro": true,              
+        "rgblight":true                  
+    }, 
+        
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x1313",
+        "vid": "0x1209",
+        "force_nkro":true 
```

Comment:
In general, this is a bad idea. Some BIOS/Firmware, and some (*nix) bootloaders don't like NKRO, and this pretty much ensures that you won't be able to use the board. 

If NKRO is eanbled, the state is saved between reboots/power cycles of the board and can be toggled via the magic keycodes. 
https://docs.qmk.fm/keycodes_magic#magic-keycodes

```suggestion
```